lib/routes/account.js

```diff
@@ -1113,13 +1115,21 @@ module.exports = function (
               push.notifyUpdate(uid, 'accountVerify')
 
               return db.verifyEmail(account)
-                .then(mailer.sendPostVerifyEmail.bind(
-                    mailer,
-                    account.email,
-                    {
-                      acceptLanguage: request.app.acceptLanguage
+                .then(
+                  function() {
+                    // Our post-verification email is very specific to sync,
+                    // so don't send it if we're sure this is not for sync.
+                    // Older clients will not send a 'service' param here
```

Comment:
"Older clients"? Is there many of them?

The idea of defaulting to true for an email that is specific to sync worries me. I feel any new service will forget to pass a `service=foo` parameter, and we'll have the same issue this is trying to resolve.